pages/services/kubernetes/1.0.0-1.9.3/limitations/index.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: layout.pug
+navigationTitle: Limitations
+title: Limitations
+menuWeight: 60
+excerpt:
+---
+
+<!-- This source repo for this topic is https://github.com/mesosphere/dcos-kubernetes -->
+
+
+Currently, the DC/OS Kubernetes package has the following limitations.
+We will be adding features rapidly as we approach General Availability (GA).
```

Comment:
approach General Availability (GA) --> shouldn't we change this?

Reply:
Done.